web/gui/README.md

```diff
@@ -1,108 +1,106 @@
-# Netdata agent web GUI
+# The standard web dashboard
```

Comment:
We're missing a link to the web dashboards overview in this file. Not sure where I'd put it, but it will be nice to tie it all together.